fortinet-fortirecon-easm/info.json

```diff
@@ -3713,6 +3720,261 @@
           "description": "(Optional) Select to retrieve linked assets. By default it is unchecked and set to False."
         }
       ]
+    },
+    {
+      "operation": "update_archived_asset",
+      "title": "Update Archived Asset",
+      "annotation": "update_archived_asset",
+      "description": "Update the given asset ID status to active. It removes the given asset from archived assets but its linked assets will still be there in the archived.Asset will be shown on the profile settings seeds section immediately. However, asset will get scanned again within 24 hours and will be shown on the active assets, if any issues found during scan those will be shown on the issues page.",
+      "category": "investigation",
+      "is_config_required": true,
+      "visible": true,
+      "enabled": true,
+      "output_schema": {
+        "message": ""
+      },
+      "parameters": [
+        {
+          "name": "asset_id",
+          "title": "Asset ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "description": "Specify the archived asset ID to update the status to active.",
+          "tooltip": "Specify the archived asset ID to update the status to active."
+        }
+      ]
+    },
+    {
+      "operation": "update_archived_issue",
+      "title": "Update Archived Issue",
```

Comment:
Title can be updated to `Mark Archived Issue as Active`.

Reply:
This is done